tools/@aws-cdk/security-guardian/README.md

```diff
@@ -0,0 +1,93 @@
+# Security Guardian
+
+A GitHub Action tool designed to validate changed AWS CloudFormation templates against custom [cfn-guard](https://github.com/aws-cloudformation/cloudformation-guard) rules. Supports both local paths and remote URLs for rule sets.
```

Comment:
I think this tool is planned to do more checks than running cfn-guard. As I understood, for now it will run cfn-guard against one rule, with a plan to extend these rules. 
Also, there is some static check logic is planned to be done on the snapshot templates, that will be done in the tool, and not by cfn-guard.